amti/clis/associate.py

```diff
@@ -29,7 +29,7 @@
     '--qual', '-q', 
     help='QualificationId (or name, if --name flag passed).')
 @click.option(
-    '--value', '-v', 
+    '--integer_value', '-v', 
```

Comment:
If we use `type=int` here (i.e. `    '--integer-value', '-v', type=int`) then click will handle validating the user input, return immediately if the input is invalid, and present a friendly error message.